### ISSUE TITLE
For those who need pretrain model，I put my pretrain model here

### ISSUE THREAD

**Issue by yak1990** [xjwxjw/Pytorch-Robust-Motion-In-betweening#10]
I put my trained model here，please checkout

https://drive.google.com/drive/folders/1o-VTFsPb5wfrZ1d-PzyT4PBLTJisjE9V?usp=sharing

**Comment by huang-1030** [comment]
> I put my trained model here，please checkout
> 
> https://drive.google.com/drive/folders/1o-VTFsPb5wfrZ1d-PzyT4PBLTJisjE9V?usp=sharing

Can you give your training hyperparameters, thanks.

**Comment by yak1990** [comment]
the hyperparameters is just like the author give ,but remove the foot slide loss
